docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Expandable.htm

```diff
@@ -277,14 +279,17 @@ <h3>Summary</h3>
                                 example:</p><p></p></span>
                             <pre name="code" class="examplecode
                             prettyprint">
+                            <p>HTML code:</p>
```

Comment:
Would you please re-check the location of this "\<p>HTML code:\</p>"?
It is included inside of \<pre>\</pre>, even it should be located outside of it.

Reply:
Done

Reply:
Done.